main.py

```diff
@@ -0,0 +1,68 @@
+import uvicorn
+from typing import List
+from opencdms.models.climsoft.v4_1_1_core import Station
+from fastapi import FastAPI, Depends
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+from sqlalchemy.orm.session import Session
+from pydantic import BaseModel
+
+app = FastAPI()
+
+# Set database url to mariadb_climsoft_db_v4 database url
+database_url = ""
```

Comment:
Can you move the database url into an environment variable so that it can be controlled from the command line (and then remove the comment that is specific to Climsoft?)